### PR TITLE
EVG-2430 parse js test results

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: self-tests.yml [hunk 1]
```diff
@@ -809,7 +814,7 @@ buildvariants:
       - ubuntu1604-build
       - ubuntu1604-test
     expansions:
-      build_env: "GOROOT=/opt/go DISABLE_COVERAGE=yes EVERGREEN_ALL=true PATH=/opt/node/bin:$PATH"
+      build_env: "GOROOT=/opt/go DISABLE_COVERAGE=yes EVERGREEN_ALL=true"
```
Comment (tychoish) [inline self-tests.yml]: did you mean to revert this change? 

I think you want to specify a karama reporter here, right?

File: makefile [hunk 2]
```diff
@@ -35,6 +35,7 @@ srcFiles := makefile $(shell find . -name "*.go" -not -path "./$(buildDir)/*" -n
 testSrcFiles := makefile $(shell find . -name "*.go" -not -path "./$(buildDir)/*" -not -path "*\#*")
 currentHash := $(shell git rev-parse HEAD)
 ldFlags := "$(if $(DEBUG_ENABLED),,-w -s )-X=github.com/evergreen-ci/evergreen.BuildRevision=$(currentHash)"
+karmaFlags := $(if $(REPORTER),--reporters $(REPORTER),)
```
Comment (tychoish) [inline makefile]: can this just be KARMA_REPORTER?

Comment (john-m-liu) [reply makefile]: changed

File: self-tests.yml (hunk 1 above)
Comment (john-m-liu) [reply self-tests.yml]: This is intentional. I had this on the wrong build variant before

**Comment by john-m-liu** [comment]
I moved the task to the ubuntu variant
